### ISSUE TITLE
Ketoxime synthesis (2022)

### ISSUE THREAD

**Issue by dmitrij176** [OpenSourceMycetoma/Series-1B-Ketoximes#1]

**Comment by MFernflower** [comment]
This was another idea I had but I do not know where to source the furnished hydroxylamine @dmitrij176 



![oxime_idea_2022](https://user-images.githubusercontent.com/3164942/148660681-6e195031-50fe-40df-9213-774a6a4b2ac3.png)

**Comment by dmitrij176** [comment]
> This was another idea I had but I do not know where to source/how to make the O-(2-dimethylaminoethyl)hydroxylamine starting material @dmitrij176
> 
> ![oxime_idea_2022](https://user-images.githubusercontent.com/3164942/148660681-6e195031-50fe-40df-9213-774a6a4b2ac3.png)

The substrate can be synthesised by means of a condensation reaction. Are there any records of Eplivanserin being tested against fungi In the past?

**Comment by MFernflower** [comment]
Eplivanserin itself has a different structure than my idea above:

https://www.targetmol.com/ss/files/structures/130579-75-8.png

Eplivanserin  itself has sparse information available online @dmitrij176

**Comment by MFernflower** [comment]
Eplivanserin itself has a different structure than my proposed compound:

![epli](https://www.targetmol.com/ss/files/structures/130579-75-8.png)

Eplivanserin has very sparse information available online @dmitrij176 so doubtful it was screened 

(reposted for inserting image)

**Comment by bebi78** [comment]
Hello, I'm happy to join the group.
I have seen that Enamine offers the dimethylaminoethylhydroxylamine. Enamine is partner of the DNDi for a Covid19 project. Maybe they will share it for free.

**Comment by MFernflower** [comment]
@bebi78 We are currently waiting on results from the in-vitro screening of the first batch of compounds @dmitrij176 


I have in the mean time proposed two compounds - if you are able to synthesize these two compounds and send them to UCL that would be amazing!!


The two compounds:

![image](https://user-images.githubusercontent.com/3164942/152608454-0da59964-dd5d-4a47-a7ae-d803f94795f8.png)

**Comment by bebi78** [comment]
@MFernflower Thanks for the suggestions. The left compound should be no problem to synthesize. I think for the right compound (dichlorophenyl derivative), Dmitrij could do this faster than me if he still has the ketone precursor. @dmitrij176 
He would just need the dimethylaminoethyl starting compound, which we might get from Enamine. We could ask Ben from DNDi or our contacts at Enamine for that starting compound if they can provide it. @bendndi 
Best,
Bernhard
---------------
Bernhard Biersack
Organische Chemie 1
Universität Bayreuth, Germany

**Comment by MFernflower** [comment]
@bebi78 I have not been in contact with Ben for a while - could you try to contact him outside of github? 

@dmitrij176

**Comment by bebi78** [comment]
@MFernflower @dmitrij176 Sure, I will contact Ben, and Tatiana of Enamine and let you know.

**Comment by bebi78** [comment]
@dmitrij176 I often apply O-THP-protected hydroxyl amine (commercially available) for the synthesis of hydroxamic acids. Might be another variation in case you don't prepare the oximes under acidic conditions.

**Comment by dmitrij176** [comment]
> @bebi78 I have not been in contact with Ben for a while - could you try to contact him outside of github?
> 
> @dmitrij176

I havent contacted him outside github as well. @mattodd would that be possible?

**Comment by bebi78** [comment]
@dmitrij176 Sergei from Enamine has provided an offer for the hydrochloride salt (see below). It is quite expensive so I asked him for a special discount as DNDi partner but haven't reveived any answer from him.  I have also attached a pdf file with the synthesis of the compound from cheap starting materials.

[ol500262y_si_001.pdf](https://github.com/OpenSourceMycetoma/Series-6-Ketoximes/files/8088736/ol500262y_si_001.pdf)


-------------------------------------
Good afternoon Bernhard,

Thank you very much for your inquiry.
Unfortunately, we do not have any of the free base, but we seem to have plenty of the corresponding dihydrochloride salt.
Please find our quote below for more details on the 2xHCl:

product ID: EN300-255113,
chemical name: O-[2-(dimethylamino)ethyl]hydroxylamine dihydrochloride, 95%, CAS 2848-78-4, MFCD09836447,
purity: 95%,
amount: 100 mg, price: EUR 230, delivery: 5-7 days,

Shipping cost is EUR 60.

Best regards,
Sergei

____________________________________
Sergei Pedan [Pronouns: He / Him / His]
Sales Manager

Enamine Ltd.
78 Chervonotkatska Street
Kyiv, 02094
Ukraine
http://www.enamine.net/
Phone: +380 44 333 56 06
e-Fax: +371 660 132 45
Fax: +380 44 537 32 53, +380 44 495 88 12
E-mail: [s.pedan@enamine.net](mailto:s.pedan@enamine.net)

**Comment by bebi78** [comment]
@dmitrij176 Yes. The chloroethyl-dimethylamine is available from Aldrich.

https://www.sigmaaldrich.com/DE/en/product/aldrich/d141208

**Comment by dmitrij176** [comment]
<img width="435" alt="image" src="https://user-images.githubusercontent.com/44785501/169874101-f186e8b6-638c-4ce4-a163-0425e8d27dc3.png">

I would like to propose two new substrates for ketoxime synthesis. The azocane would be a non-standard reagent, giving a totally novel analog.

**Comment by MFernflower** [comment]
I like the azocane idea

**Comment by bebi78** [comment]
@dmitrij176 Yes, the morpholine is not far away from the modified piperazines of the fenarimol series.

**Comment by dmitrij176** [comment]
![Ketoxime library (finalised assay list)  jpeg](https://user-images.githubusercontent.com/44785501/216826164-74d9081f-9f29-4889-9fbf-24053fcd529e.jpeg)

Finalised list of ketoximes coming in the next assay batch. MYOS code update (in the Master List) with logD calculations coming soon.
